bridgemetadata/convert.py

```diff
@@ -98,4 +95,20 @@ def _hasTag(tag):
         elif "19115" in schemaName:
             return Format.ISO19139
     else:
-        return Format.FGDC
+        print ("Unknown source format")
+        return None
```

Comment:
Why not throw an `UnknownFormatException` here?
Then we can also get rid of [this piece of code](https://github.com/pvgenuchten/bridge-metadata/blob/65853022b08a849dd610b940713c567959936574/bridgemetadata/convert.py#L52-L53).